ui/dashboardApp/layout/main/Sider/index.tsx

```diff
@@ -140,7 +140,15 @@ function Sider({
     if (defaultCollapsed) {
       return []
     } else {
-      return ['debug', 'experimental']
+      let openKeys = ['debug', 'experimental']
+      const urlHash = window.location.hash.substring(2)
+      if (
+        urlHash.startsWith('instance_profiling') ||
+        urlHash.startsWith('continuous_profiling')
+      ) {
+        openKeys.push('profiling')
+      }
+      return openKeys
```

Comment:
Is it possible to just open everything? In this way we can reduce the amount of these hard codings.

Reply:
Yep, sure, it's much simpler, just `return ['debug', 'experimental', 'profiling']`, I have tested. I was just afraid it makes the left side menu too long.

Reply:
Oh, you mean we can remove `debug`, `experimental` these hard codes as well?